keyboards/keychron/x0/info.json

```diff
@@ -0,0 +1,74 @@
+{
+    "manufacturer": "Keychron",
+    "keyboard_name": "Lemokey X0",
+    "maintainer": "lalalademaxiya1",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 4096,
+            "logical_size": 2048
```

Comment:
```suggestion
            "backing_size": 4096
```

Reply:
Is 2048 the default value for `logical_size`?

Reply:
https://docs.qmk.fm/drivers/eeprom#wear_leveling-efl-driver-configuration

| config.h override |Default|
|-|-|
|#define WEAR_LEVELING_LOGICAL_SIZE |	(backing_size/2) |